### ISSUE TITLE
Task: Implement the Score Function

### ISSUE THREAD

**Issue by NimetaZ** [PhilipC6/Kujira-ISP#40]

**Comment by NimetaZ** [comment]
finished creating a function, now have to make the function tangible with a variable score rather than a set one

**Comment by NimetaZ** [comment]
functions working, now waiting on a combined test with more/all merges. will create a new task to create the score object (display in game)
